packages/gatsby/src/redux/__tests__/pages.ts

```diff
@@ -83,85 +85,98 @@ describe(`Add pages`, () => {
   })
 
   it(`adds an initial forward slash if the user doesn't`, () => {
-    const action = actions.createPage(
+    const actionsArray = actions.createPage(
       {
         path: `hi/`,
         component: `/whatever/index.js`,
       },
       { id: `test`, name: `test` }
     )
+    const action = actionsArray.filter(a => a.type === `CREATE_PAGE`)[0]
```

Comment:
since all these are only passed to the pages reducer, we filter out the `CREATE_NODE` action

Reply:
NIT, but instead of `.filter(<predicate>)[0]` you can use just `.find(<predicate>)`